### DISCUSSION TITLE
"Quick-Link" bookmarks

### DISCUSSION THREAD

**Discussion by pijusz** [dailydotdev/daily#542] [Feature Requests]
Would love to see standard links I go to, which could be added manually or gathered from use history. Firefox default home page has 10+ links that I can quickly go to. F.e.: Youtube, twitter, facebook etc.

This could be added to sidebar or as smaller cards at the top of dailydev body.

**Comment by rebelchris** [comment]
Hi @pijusz I think we have exactly what you are looking for: [Shortcut links](https://twitter.com/dailydotdev/status/1498657307345010693?s=20&t=NBES7-vZS6cQz4mIryl0sA)
